Source/ee/Vpu.h

```diff
@@ -39,7 +40,17 @@ class CVpu
 							CVpu(unsigned int, const VPUINIT&, CGIF&, uint8*, uint8*);
 	virtual					~CVpu();
 
-	void					Execute(bool);
+	void* operator new(size_t allocSize)
+	{
+		return framework_aligned_alloc(allocSize, 0x10);
```

Comment:
Is aligned allocation necessary here? In Ee::CSubsystem, it's needed because the MIPS contexts that are stored there are referenced by jitted code (which uses aligned SIMD reads), while nothing from CVpu is referenced from it. 

I did a quick test on my side and aligned allocs didn't seem to be necessary.